src/platform/utilities/tests/authentication.unit.spec.js

```diff
@@ -12,17 +12,18 @@ import {
 
 import { mockApiRequest, resetFetch } from '../../testing/unit/helpers';
 
-let windowOpen;
 let oldWindow;
+let oldOpen;
 
 const fakeWindow = () => {
   oldWindow = global.window;
-  windowOpen = sinon.stub().returns({
+  oldOpen = global.open;
+  global.open = sinon.stub().returns({
```

Comment:
It's recommended to use global instead of window per `https://stackoverflow.com/a/41888736` and `https://github.com/facebook/jest/issues/3692#issuecomment-408132529`.